keyboards/zlant/keymaps/default/keymap.c

```diff
@@ -20,3 +20,27 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   )
 
 };
+
+
+/*  Base
+ * ,-----------------------------------------------------------------------------------.
+ * | TAB  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | LCTRL|   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | ESC  |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |LED H+|LED H-| GUI  | Alt  | SHIFT|    Space    |Raise | Left | Down |  Up  |Right |
```

Comment:
```suggestion
 * |LED H+|LED H-| LGUI | LAlt |LShift|    Space    |  Fn  | Left | Down |  Up  |Right |
```
Some of this capitalization is a bit odd. I suggest using: `Tab`, `LCtrl`, `Esc`, `PgUp`, `PgDn`, `PScr`.
Also: `Bspc` instead of `Bksp`, `LGUI` instead of `GUI`, `LAlt` instead of `Alt`, `LShift` instead of `Shift`. That way the labels will be more in line with actual QMK keycodes.